rollup.config.js

```diff
@@ -38,6 +38,7 @@ const rollup = {
 			return chunkInfo.name.replace('build/__snapshots__/', '') + '.js';
 		},
 	},
+	context: 'window',
```

Comment:
Avoid this warning. https://rollupjs.org/troubleshooting/#error-this-is-undefined
<img width="931" alt="image" src="https://github.com/user-attachments/assets/f5768a19-e19d-4698-a09f-d8d57be86b58">

Reply:
We should add a link to https://github.com/sindresorhus/memoize/issues/102 so we will drop this after the change is done. 
```suggestion
   // TODO: blah blah
	context: 'window',
```